src/main/java/mods/railcraft/world/level/block/CokeOvenBricksBlock.java

```diff
@@ -51,26 +47,24 @@ public BlockEntity newBlockEntity(BlockPos blockPos, BlockState blockState) {
     return new CokeOvenBlockEntity(blockPos, blockState);
   }
 
-  @Nullable
-  @Override
-  public <T extends BlockEntity> BlockEntityTicker<T> getTicker(Level level, BlockState blockState,
```

Comment:
Does this not need ticking anymore?

Reply:
oh, so that's how new ticking events works, oops